enterprise/reporting/error_reporting.go

```diff
@@ -505,16 +520,23 @@ func (edr *ErrorDetailReporter) aggregate(reports []*types.EDReportsDB) []*types
 				ReportedAt: firstReport.ReportedAt * 60 * 1000,
 			},
 		}
-		var errs []types.EDErrorDetails
-
-		reportsCountMap := lo.CountValuesBy(reports, func(rep *types.EDReportsDB) types.EDErrorDetails {
-			return types.EDErrorDetails{
+		messageMap := make(map[string]int)
```

Comment:
why CountValuesBy is removed any reason for it?

Reply:
To include count on `reportsCountMap` while grouping on reports we need to consider count as an argument which is unnecessary increasing the set. So apart from using that we used the normal for loop and increased the count.